junit-jupiter-api/src/templates/resources/main/org/junit/jupiter/api/condition/JRE.java.jte

```diff
@@ -193,7 +150,13 @@ public enum JRE {
 	 */
 	@API(status = STABLE, since = "5.12")
 	public static JRE currentJre() {
-		return CURRENT_JRE;
+		return switch (CURRENT_VERSION) {
+			case UNDEFINED_VERSION -> UNDEFINED;<%--
```

Comment:
Similarly here, `CURRENT_VERSION` can never be `UNDEFINED_VERSION`, so we should likely remove that case and update the Javadoc.

Or am I missing something?

In other words, do we still need to support the `UNDEFINED_VERSION` across all of these methods now that we are using `Runtime.version().feature()` instead of parsing the `java.version` JVM system property?

Reply:
Done in 9ac1c76da531ab463952f1951409581fbb9a052e